registry/repository.go

```diff
@@ -55,7 +56,7 @@ func (c *RepositoryClient) ListSignatureManifests(ctx context.Context, manifestD
 	// TODO(shizhMSFT): filter artifact type at the server side
 	if err := c.Repository.Referrers(ctx, ocispec.Descriptor{
 		Digest: manifestDigest,
-	}, func(referrers []artifactspec.Descriptor) error {
+	}, "", func(referrers []artifactspec.Descriptor) error {
```

Comment:
Instead of empty artifact type, you may specify the artifact type when invoking `Referrers()`.

Reply:
Gotcha, updated to `ArtifactTypeNotation`